SU2_CFD/src/drivers/CMultizoneDriver.cpp

```diff
@@ -239,12 +239,12 @@ void CMultizoneDriver::Preprocess(unsigned long TimeIter) {
 
     /*--- Set the initial condition for EULER/N-S/RANS ---------------------------------------------*/
     /*--- For FSI, the initial conditions are set, after the mesh has been moved. --------------------------------------*/
-    if (!fsi && !config_container[iZone]->GetDiscrete_Adjoint() && config_container[iZone]->GetFluidProblem()) {
+    if (!fsi && config_container[iZone]->GetFluidProblem()) {
```

Comment:
I don't think the multizone driver is going to be recorded anytime soon so this was redundant.